src/camt/types.ts

```diff
@@ -114,6 +120,19 @@ export interface Transaction {
   returnAdditionalInformation?: string;
 }
 
+export interface BankTransactionCode {
+  /** Specifiesthe business area of the underlying transaction. */
+  domainCode?: string;
+  /** Specifies the family within the domain of the underlying transaction.  */
+  domainFamilyCode?: string;
+  /** Specifies the sub-product family within a specific family of the underlying transaction. */
+  domainSubFamilyCode?: string;
+  /** Bank transaction code in a proprietary form, as defined by the issuer. */
+  proprietaryCode?: string;
+  /** Identification of the issuer of the proprietary bank transaction code. */
+  proprietaryCodeIssuer?: string;
+}
```

Comment:
Can we move this to outside of BankTransactionCode? 

so just proprietaryCode / proprietaryCodeIssuer

Reply:
We can, but where should we move it? It's actually the proprietaryCode of the `BankTransactionCode`. This is provided when the bank chooses not to provide a standard code as specified in the document in the other message, for instance [here](https://github.com/moscar09/iso20022.js/blob/f4b0ea33198b7966564d83b32034efa258a87740/test/assets/goldman_sachs/camt_053_us_v2_sample.xml#L726)

Reply:
You're right.. Ntry's have proprietaryCodes AND BankTransactionCodes themselves have more proprietary codes